tests/utils.test.js

```diff
@@ -154,12 +149,24 @@
 			for (const [range, string] of definition) {
 				for (const time of range) {
 					assert.strictEqual(
-						timeDelta(new unmockedDate(time), true),
+						timeDelta(new sb.Date(time), true),
 						string,
 						"Input: " + time
 					);
 				}
 			}
 		});
+
+		it("shows leap years as 1y", () => {
```

Comment:
As mentioned before, this behaviour should be toggleable, and as such should be included in tests with both being turned on and off.